src/app/map/_components/MapMenu.tsx

```diff
@@ -8,7 +8,7 @@ type MapMenuProps = {
 
 export function MapMenu({ floor, setFloor }: MapMenuProps) {
   return (
-    <div className="absolute left-10 bottom-10 p-4">
```

Comment:
the more you use absolute the more messy your design will get and you'll end up fighting with z-indexes all over the place